### DISCUSSION TITLE
http 和 ssh协议进行git操作的地址是引用的同一个变量吗？

### DISCUSSION THREAD

**Discussion by karypi** [OpenCSGs/csghub#598] [Q&A]

**Comment by karypi** [comment]
STARHUB_SERVER_FRONTEND_URL    csghub-server 里的这个变量是 http 拉取代码的地址么

**Comment by pulltheflower** [comment]
> STARHUB_SERVER_FRONTEND_URL csghub-server 里的这个变量是 http 拉取代码的地址么

@karypi 是的，这个是前端项目的启动地址，我们会通过这个地址来拼接 git 仓库的 clone url

**Comment by karypi** [comment]
> > STARHUB_SERVER_FRONTEND_URL csghub-server 里的这个变量是 http 拉取代码的地址么
> 
> @karypi 是的，这个是前端项目的启动地址，我们会通过这个地址来拼接 git 仓库的 clone url

谢谢，SSH 协议是这个变量 GITEA_SSH_DOMAIN 吗？

**Comment by Rader** [comment]
> > > STARHUB_SERVER_FRONTEND_URL csghub-server 里的这个变量是 http 拉取代码的地址么
> > 
> > 
> > @karypi 是的，这个是前端项目的启动地址，我们会通过这个地址来拼接 git 仓库的 clone url
> 
> 谢谢，SSH 协议是这个变量 GITEA_SSH_DOMAIN 吗？

是的
